### DISCUSSION TITLE
Using flameshot as an editor for the image in the clipboard?

### DISCUSSION THREAD

**Discussion by offray** [flameshot-org/flameshot#2488] [Q&A]
Hi all. Thanks for flameshot. I use it and recommend it for my classes to share screenshots with my students and edit them when I'm making some explanation, using numbers, arrows and boxes to clarify my explanation. 

I would like to use Flameshot as a "image clipboard editor" when screenshots come from the web browser. In that use case, I use the "take full page screenshot" shortcut from the web browser, to put in my screenshot pages or the web pages, that are not shown in the screen  and I would like to be able to "paste" the web page screenshot in Flameshot to edit it (adding numbers, boxes, arrows to it) before posting it online. There is any way to do this?

**Comment by mmahmoudian** [comment]
Duplicate of:

- #2005
- #1100
- #240
- #83

**Reply by shollingsworth** [reply]
FYI For those searching for this, for existing images I found https://github.com/ksnip/ksnip has a lot of the cool annotation features flameshot does.
